tests/shtest

```diff
@@ -165,13 +165,21 @@ cmp $d/out $d/expected
 printf "[1,2][3,4]\n" | $JQ -cs add > $d/out 2>&1
 cmp $d/out $d/expected
 
-# Regression test for -0 / --nul-output
+# Regression test for --raw-output0
 printf "a\0b\0" > $d/expected
-printf '["a", "b"]' | $JQ -0 .[] > $d/out 2>&1
```

Comment:
Do we want to keep a `-0` test but now that it's not an option?

Reply:
I'll leave that up to @itchyny.  I don't think there's a need for a test `-0` is now not an option.

Reply:
I don't think it's important to test that.